worlds/aquaria/Regions.py

```diff
@@ -846,6 +899,9 @@ def connect_regions(self) -> None:
         self.__connect_abyss_regions()
         self.__connect_sunken_city_regions()
         self.__connect_body_regions()
+        if (options.objective.value == Objective.option_killing_the_four_gods or
+                options.objective.value == Objective.option_gods_and_creator):
```

Comment:
This combination of option checks (or its negation) are used pretty often. Maybe it should be assigned to an attribute.